apps/console/src/features/roles/components/edit-role/edit-role-federated-groups-accordion.tsx

```diff
@@ -0,0 +1,449 @@
+                removedGroups.push(group);
+            }
+        });
+
+        setSelectedGroupsOptions([
+            ...selectedGroupsOptions,
+            ...removedGroups
+        ]);
+    };
+
+    /**
+     * Resolves the IDP image.
+     * 
+     * @returns Resolved IDP image.
+     */
+    const resolveIDPImage = (): string => {
+        if (identityProvider.image) {
+            return ConnectionsManagementUtils.resolveConnectionResourcePath(
+                "", identityProvider.image);
+        }
+        
+        return getConnectionIcons().default;
+    };
+
+    return (
+        <EmphasizedSegment
+            className="mt-2"
+            basic
+        >
+            <Accordion
+                onChange={ onAccordionChange }
+                elevation={ 1 }
+                variant="elevation"
+                disableGutters
+                expanded={ isExpanded }
+            >
+                <AccordionSummary
+                    aria-controls="panel1a-content"
+                    id="panel1a-header"
+                    expandIcon={ <ChevronDownIcon /> }
+                >
+                    <>
+                        <GenericIcon
+                            square
+                            transparent
+                            className="card-image"
+                            size="mini"
+                            icon={ resolveIDPImage() }
+                            floated="left"
+                        />
+                        <Typography
+                            className="ml-4"
+                            variant="h6"
+                        >
+                            { identityProvider.name }
+                        </Typography>
+                    </>
+                    
+                </AccordionSummary>
+                <AccordionDetails>
+                    <>
+                        {
+                            isReadOnly
+                                ? (
+                                    <Autocomplete
+                                        multiple
+                                        disableCloseOnSelect
+                                        options={ selectedGroupsOptions ? selectedGroupsOptions : [] }
+                                        value={ selectedGroupsOptions ? selectedGroupsOptions : [] }
+                                        getOptionLabel={ (group: IdentityProviderGroupInterface) => 
+                                            group.name }
+                                        renderInput={ (params: AutocompleteRenderInputParams) => (
+                                            <TextField
+                                                { ...params }
+                                                placeholder= { t("console:manage.features.roles.edit.groups" + 
+                                                        ".actions.search.placeholder") }
```

Comment:
Let's fix indentation here.

```suggestion
                                                    ".actions.search.placeholder") }
```

Reply:
Addressed 5ce5a869d09cb34de9eac44ddee5d1053e6006a4